openqabot/approver.py

```diff
@@ -165,7 +165,11 @@ def was_ok_before(self, failed_job_id: int, inc: int) -> bool:
             return False
 
         current_build = older_jobs["data"][0]["build"][:-2]
-        current_build_date = datetime.strptime(current_build, "%Y%m%d")
+        try:
+            current_build_date = datetime.strptime(current_build, "%Y%m%d")
+        except (ValueError, TypeError):
+            log.info("Could not parse build date %s", current_build)
```

Comment:
I suggest to make it clear what the consequence is. "Could not parse build date", so what? If it does not have any effect why parse the build date at all? If it does have an impact, what will it be? Will it fallback to a default value?